zql/zql.peg

```diff
@@ -387,8 +387,10 @@ top
 procLimitArg
   = _ "-limit" _ limit:unsignedInteger { RETURN(limit) }
 
+cutArg = (_ "-c" { RETURN(makeArg("c", NULL)) })*
```

Comment:
why the * ?

Reply:
This  is done in other args... for example `sort -r -r -r` is gramatically valid but rejected in the parser-support builders. I'm just following the pattern and if we think this should be changed i don't mind but wholesale...

Reply:
Right, this was because sort accepts a few different arguments.  The parser used to have explicit rules for the different orderings of the different args, but there's a combinatorial issue there as new arguments are added.
Anyway, agreed that we can just keep the pattern here...

Reply:
yeah, i think anything that avoids our inscrutable parser errors is a win. (separately, i think `-x -x -x` should be handled just like `-x` and shouldn't be an error... but getting off topic).